### DISCUSSION TITLE
We sent a reveiw request in Korean to tensorflow on gitlocalize. I'd like to ask when can you check it out?

### DISCUSSION THREAD

**Discussion by kimdoyeon817** [tensorflow/docs-l10n#572] [Q&A]
We sent a reveiw request in Korean to tensorflow on gitlocalize. I'd like to ask when can you check it out?

**Comment by ilyaspiridonov** [comment]
@kimdoyeon817 Thank you for your submission. We are delivering a large portion of translations early next week, your RR will be reviewed with this batch.
